hapi-proto/src/main/proto/ScheduleService.proto

```diff
@@ -29,7 +29,26 @@ import "Response.proto";
 import "TransactionResponse.proto";
 import "Transaction.proto";
 
-/* Transactions and queries for the Schedule Service */
+/*
+Transactions and queries for the Schedule Service
+
+The Schedule Service allows transactions to be submitted without all the required signatures and provide functionality for anyone to provide the required signatures independently after a transaction was created.
+
+Execution:
+Scheduled Transactions are executed once all required signatures are collected and witnessed. Every time new signature is provided, a check is performed on the "readiness" of the execution.
+The scheduled transaction will be executed immediately after the transaction that triggered it and will be externalised in a separate Transaction Record.
+
+Transaction Record:
+The timestamp of the scheduled transaction will be equal to consensusTimestamp + 1 nano, where consensusTimestamp is the timestamp of the transaction that triggered the execution.
+The Transaction ID of the scheduled transaction will have scheduled set to true and inherit the transactionValidStart and accountID from the Schedule Create transaction.
```

Comment:
Suggestion: "Schedule Create "->" **ScheduleCreate** " I noticed that in some places, naming is written with spaces and in others - without.